src/plugins/usrpfp/index.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { definePluginSettings } from "@api/Settings";
+import { Link } from "@components/Link";
+import { Devs } from "@utils/constants";
+import definePlugin, { OptionType } from "@utils/types";
+import { User } from "discord-types/general";
+
+import { DataFile } from "./types";
+
+const BASE_URL = "https://userpfp.github.io/UserPFP/source/data.json";
+
+let data: DataFile = {
+    avatars: {},
+    badges: {},
+};
+
+const settings = definePluginSettings({
+    preferNitro: {
+        description:
+            "Which avatar to use if both Nitro and UserPFP avatars are present",
```

Comment:
this name is slightly confusing. maybe something like this instead?
```suggestion
            "Which avatar to use if both default animated (Nitro) pfp and UserPFP avatars are present",
```

Reply:
Replaced with your suggested modification